File: 01-02-2024/Question/Python Solutions/Solution2.py
cpi = 18
cpe = 12
cost = 0

ni = int(input()) 
ne = int(input())

if(ni<0 or ne<0):
    print("INVALID INPUT")

elif(ni==0 and ne==0):
    print("Total estimated Cost : 0.0 INR")

else:
    for i in range(ni):
        temp=float(input())
        cost+= cpi*temp

    for i in range(ne):
        temp=float(input())
        cost+= cpe*temp
        
    print("Total estimated Cost : %.1f INR"%cost)

File: 01-02-2024/Question/Cpp Solutions/Solution2.cpp
#include <stdio.h>

int main()
{
    int ni, ne, i;
    float temp, cost = 0.0, cpi = 18, cpe = 12;
    scanf("%d %d", &ni, &ne); // 6 3
    for (i = 0; i < ni; i++)
    {
        scanf("%f", &temp); // 12.3,15.2..
        cost = cost + temp * cpi;
    }
    for (i = 0; i < ne; i++)
    {
        scanf("%f", &temp); // 10.10,10.00..
        cost = cost + temp * cpe;
    }
    printf("Total estimated Cost:%.1f INR", cost);
}